### ISSUE TITLE
Real world benchmarks

### ISSUE THREAD

**Issue by armintoepfer** [eldariont/svim#6]
Hi!

I encourage you to use the latest HG002 datasets to investigate your FP and FN numbers using `truvari` and GIAB annotation.

PacBio offers 
 - `15kb` library with CCS: ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/PacBio_CCS_15kb/
 - `>15kb` long read library: https://downloads.pacbcloud.com/public/dataset/HG002/Sequel-201810/

I benchmarked `svim` internally and didn't get great results. Just to give you a feeling, both public versions of `sniffles` and `pbsv` achieve `>0.90` F1 score.

Happy developing!
Armin

**Comment by eldariont** [comment]
Hi Arnim,

great, I haven't been aware of these annotations :) Thanks for the links! I'll run SVIM on the data and see which results I get.

Many thanks
David

**Comment by armintoepfer** [comment]
Since GIAB is not DUP aware, it would be nice to limit prediction only on insertions and deletion.
I personally know that adding DUPs will decrease recall and specificity.

Let me know what results you get.

**Comment by armintoepfer** [comment]
Maybe, it would be useful to (i) have better defaults flagging low-quality variants as not passing, (ii) add the genotype column so that common tools can parse the VCF, and (iii) make it importable for IGV.

Feel free to ignore the following, just an advice. For your paper, I'd recommend using truvari and the GIAB annotation for your tool comparisons. Because this is a third-party tool that gives objective results. I honestly don't trust comparison, when the author also writes the tool to compare :)

**Comment by eldariont** [comment]
Thanks for the good suggestions.

(i) SVIM already outputs FILTER flags for calls with a score lower than 30 (and lower than 20 for inversions). It's hard to decide what "better defaults" could be because it really depends on the input read coverage. Calls lower than 30 are usually very questionable which is why they have the FILTER flag set.
(ii) I added the genotype column with 87b06cc4ed13c8dfc892e3d08766846d7dc1eeb2
(iii) With 87b06cc4ed13c8dfc892e3d08766846d7dc1eeb2, I also changed the vcf version to 4.2 in order to ensure compatability with the current IGV version

You're probably right about the dangers of developers who write their own comparison tool 😄 I will use truvari or another independent tool whenever I can.

**Comment by eldariont** [comment]
I looked into the FPs and FNs a bit more and found a bug in SVIM that caused the removal of correct INS calls (fixed by 885e63e5d640302458ed8bbda51b76c86c987fd6). I also fixed the VCF output further to follow the VCF specs more closely.

When altering SVIM's parameters, I found that a higher distance threshold of 1.4 for the signature clustering leads to better results than those I posted above:

```
Run        F1      Precision  Recall  FP+FN
svim_eval  0.9318  0.9336     0.9301  1312
```

I also added a wiki page describing the evaluation procedure: https://github.com/eldariont/svim/wiki/Evaluation-on-HG002-with-GiaB-draft-truth-set.
